packages/ifpack2/src/Ifpack2_ContainerFactory_decl.hpp

```diff
@@ -48,6 +48,7 @@
 #include "Ifpack2_DenseContainer.hpp"
 #include "Ifpack2_SparseContainer.hpp"
 #include "Ifpack2_BandedContainer.hpp"
+#include "Ifpack2_BlockTriDiContainer.hpp"
```

Comment:
Why does the ContainerFactory's decl file need to include all the Container subclasses' header files?  Shouldn't those go in the ContainerFactory's def file?

Reply:
Sorry @ambrad never mind :)

Reply:
@ambrad @mhoemmen This was my fault. I can make this change now that this is merged.

Reply:
Great!

Reply:
Sorry it took a while but I just checked in the fix.